### ISSUE TITLE
Bug - Re-enable uploading a11y testing for Patternfly

### ISSUE THREAD

**Issue by dlabaj** [patternfly/patternfly#7885]
**Describe the problem**
Currently a11y testing is disable/commented out in the github action to upload it to surge. This should be re-enabled after the 6.4.0 release. 

The code that is disabled can be found here:

[https://github.com/patternfly/patternfly/blob/df25cbdd337e39355605de535c2a9d80c6c3bfe6/.github/workflows/release.yml#L62](https://github.com/patternfly/patternfly/blob/df25cbdd337e39355605de535c2a9d80c6c3bfe6/.github/workflows/release.yml#L62)

---

**Jira Issue:** [PF-2750](https://redhat.atlassian.net/browse/PF-2750)
